pytoml/writer.py

```diff
@@ -87,10 +90,19 @@ def _format_value(v):
             return v.strftime('%Y-%m-%dT%H:%M:%S') + suffix
     elif isinstance(v, list):
         return _format_list(v)
+    elif _isinstance_purepath(v):
+        return '"{}"'.format(v)
```

Comment:
The string absolutely requires escaping, especially on Windows there'll be quite a few backslashes.

```suggestion
        return _escape_string(str(v))
```

Please include tests for this.